app/helpers/teams_helper.rb

```diff
@@ -1,6 +1,11 @@
 module TeamsHelper
 
-  def conference_exists_for(team)
-    team&.conference_preference&.persisted? && team&.conference_preference&.first_conference_id.present? || team&.conference_preference&.second_conference_id.present?
+  def conference_exists_for?(team)
+    conference_preference = team.conference_preference
+    return false unless conference_preference.present? && conference_preference.persisted?
```

Comment:
Way easier to understand now I'd say 💪 
Maybe shorten this line to:
```ruby
return false unless conference_preference&.persisted?
```
But that's just personal preference, you can also leave it as it is - there's no change in behavior.